tests/NovelRT.Tests/Interop/Ecs/NrtCatalogueTest.cpp

```diff
@@ -29,7 +29,7 @@ class InteropCatalogueTest : public testing::Test
         entityCache = Nrt_EntityCache_Create(1);
         catalogue = Nrt_Catalogue_Create(0, componentCache, entityCache);
         int32_t intDeleteState = -1;
-        size_t sizeTDeleteState = -1;
+        size_t sizeTDeleteState = 1;
```

Comment:
I'm pretty sure this is going to break something. We use +1 for size_t because it wraps around to max and for a sake of a test this was easier than just using the numeric limits stuff.

Can we revert or update this please?

Reply:
> Also generic question from me - what's with all the formatting changes?

Not sure what you mean by formatting, but basically with the new options I ran in 3 kinds of issues in the tests:
1. unused variables/parameters
2. implicit conversions (e.g. `-1` in `size_t`)
3. unused returned value of `nodiscard` functions

Reply:
There are changes where you've changed the formatting of certain lines, when there wasn't a need to, from what I can see. I'll flag some of it.